src/components/common/program-section-templates/faq-program-section/FaqProgramSection.module.scss

```
@@ -0,0 +1,136 @@
+@use '@/assets/sass/variables/colors' as c;
+@use '@/assets/sass/mixins/typography.mixins' as type;
+@import '../shared/BaseSectionTemplate.module.scss';
+
+.faq-section {
+    @extend .section-container;
```

Stylelint (17.3.0)

[error] Expected a placeholder selector (e.g. %placeholder) to be used in `@extend` (scss/at-extend-no-missing-placeholder)

(scss/at-extend-no-missing-placeholder)